packages/-ember-data/tests/integration/record-data/record-data-test.ts

```diff
@@ -0,0 +1,310 @@
+import { get } from '@ember/object';
+import { setupTest } from 'ember-qunit';
+import Model from 'ember-data/model';
```

Comment:
why are we using the legacy imports instead of the packages imports?

Reply:
Copied/consistency with the other tests, we can run a codemod once ready